webpack.dev.config.js

```diff
@@ -51,6 +51,30 @@ function getAliasesForLocalDependencies(env) {
   return aliases;
 }
 
+/**
+ * Returns loader for 'ts' and 'tsx' files.
+ * If we are using '--deck' env variable we should avoid TypeScript issues from deck.gl folder.
+ * TODO: We should always use ts-loader once all issues with deck.gl types have been resolved.
+ */
+function getLoaders(env) {
+  // Use simple babel loader without typescript to avoid local deck.gl typing issues.
+  if (env["deck"]) {
```

Comment:
I had the issue with "loaders" as well. When I try to add new options to Tileset3D there type errors occur.

Reply:
> I'm not sure we need these changes. Despite errors the application starts and works after I close overlapping panel with errors. ![image](https://user-images.githubusercontent.com/18549629/154226187-5f78220c-2669-450d-9f92-971932e9733f.png) I guess it is sufficient for development.

If you will continue use applications with these errors and try to add some changes to deck.gl locally application won't update with these changes.

Reply:
Also I think we should do the same changes for loaders in dev webpack config.